.travis.yml

```diff
@@ -5,7 +5,15 @@ _environments:
   - &gcc-warnings
     - CFLAGS="-std=c89 -Wall -Wextra -pedantic"
 
+_scripts:
+  - &install_cmake35
+    - >
+        wget https://cmake.org/files/v3.5/cmake-3.5.2-Linux-x86_64.tar.gz &&
+        sudo tar xfC $(basename $_) / --strip-components=1
+
```

Comment:
Use of `sudo` here doesn't come with zero cost, falling back to standard infrastructure rather than what is reported/observed as [faster container-based VM](https://docs.travis-ci.com/user/workers/container-based-infrastructure/).

Is it worth it? It seems so, especially with this new use of `repeat-until-fail`

We can configure some environments to respond quickly (<1min feedback is common in Travis on a Pull Request), and then allow additional environments configured as below to continue with more strenuous tests.

A curiosity:

3.5 downloaded: https://travis-ci.org/nanomsg/nanomsg/jobs/124985033#L168
3.2 reported: https://travis-ci.org/nanomsg/nanomsg/jobs/124985033#L275